Jolt/Core/Core.h

```diff
@@ -180,6 +180,13 @@
 	#define JPH_VECTOR_ALIGNMENT 16
 	#define JPH_DVECTOR_ALIGNMENT 32
 	#define JPH_DISABLE_CUSTOM_ALLOCATOR
+#elif defined(__e2k__)
+	// Elbrus e2k architecture
+	#define JPH_CPU_E2K
+	#define JPH_CPU_ADDRESS_BITS 64
+	#define JPH_USE_SSE
```

Comment:
It is safer to first check the availability of the SSE macro, and then enable it.
```
#if defined(__SSE__) && !defined(JPH_USE_SSE)
 #define JPH_USE_SSE
#endif
```
You can do the same with the rest of the macros (SSE4_1, SSE4_2, AVX, AVX2, F16C, LZCNT, BMI).